RegionXComponents/src/components/RegionCard/RegionCard.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './RegionCard.scss';
+import RegionCardHeader from './RegionCardHeader/RegionCardHeader';
+import RegionCardTasks from './RegionCardTasks/RegionCardTasks';
+import CardImageItem from '../../assets/images/CardImage.png';
```

Comment:
This is a mock image. We will probably use an ident icon for the regions as well.

Reply:
In this PR we can leave as is.